README.md

```diff
@@ -98,7 +116,8 @@ function initAnswers() {
     apiKey: '<API_KEY_HERE>',
     // Required, the key used for your Answers experience
     experienceKey: '<EXPERIENCE_KEY_HERE>',
-    // Required, initialize components here, invoked when the Answers component library is loaded/ready
+    // Required, initialize components here, invoked when the Answers component library is loaded/ready.
+    //   Components may instead be initialized after the init promise resolves.
```

Comment:
since this isn't required anymore, remove the "Required" and maybe reword this slightly